dvc/output/base.py

```diff
@@ -53,6 +53,15 @@ def __init__(self, match):
         super().__init__(f"Path '{match.file}' is ignored by\n{lines}")
 
 
+class OutputStoreCacheIncompatible(DvcException):
+    def __init__(self, path):
+        msg = f"""Incompatible options for output '{path}'.
+    The dvc file specifies an output with both "cache == False" and
+    "store == True" which is not possible to satisfy.
+    """
```

Comment:
Looks like the indentation is off but not sure.

Reply:
I think that's right: I copied the other examples I saw. That'll produce:

```
Incompatible options for output 'path'.
    The dvc file specifies an output with both "cache == False" and
    "store == True" which is not possible to satisfy.
```